<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>convert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFolder">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>folder</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>video</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>audio</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="videoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>611</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>601</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="MPFourRadioButton">
          <property name="text">
           <string>MP4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="AVIRadioButton">
          <property name="text">
           <string>AVI</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QRadioButton" name="MOVRadioButton">
          <property name="text">
           <string>MOV</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="MKVRadioButton">
          <property name="text">
           <string>MKV</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QRadioButton" name="WebMRadioButton">
          <property name="text">
           <string>WebM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="FLVRadioButton">
          <property name="text">
           <string>FLV</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="WMVRadioButton">
          <property name="text">
           <string>WMV</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ThreeGPRadioButton">
          <property name="text">
           <string>3GP</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="audioWidget" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>611</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>601</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="MPThreeRadioButton">
          <property name="text">
           <string>MP3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="AACRadioButton">
          <property name="text">
           <string>AAC</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QRadioButton" name="WAVRadioButton">
          <property name="text">
           <string>WAV</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="OGGRadioButton">
          <property name="text">
           <string>OGG Vorbis</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QRadioButton" name="FLACRadioButton">
          <property name="text">
           <string>FLAC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="WMARadioButton">
          <property name="text">
           <string>WMA</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QRadioButton" name="AIFFRadioButton">
          <property name="text">
           <string>AIFF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="AMRRadioButton">
          <property name="text">
           <string>AMR</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
